app/main.py

```diff
@@ -34,13 +34,22 @@ def __truediv__(self, other):
             return Distance(round(self.km / other, 2))
 
     def __lt__(self, other):
-        return self.km < other
+        if isinstance(other, Distance):
+            return self.km < other.km
+        if isinstance(other, (int, float)):
+            return self.km < other
```

Comment:
You may be sure that you always get one of `int`, `float`, `Distance`, so you can rewrite without a double check (not only here, check for this full file)